src/renderer/coremods/commands/commands.ts

```diff
@@ -0,0 +1,382 @@
+import { CommandManager } from "../../apis/commands";
+import { ApplicationCommandOptionType, RepluggedCommand } from "../../../types";
+import { plugins, themes } from "@replugged";
+const command = new CommandManager();
+
+const commands: RepluggedCommand[] = [
+  {
+    name: "enable",
+    description: "Enable a plugin or theme",
+    usage: "/enable <plugin/theme>",
+    options: [
+      {
+        name: "addon",
+        description: "Choose the addon you want to enable",
+        type: ApplicationCommandOptionType.String,
+        required: true,
+        get choices() {
+          const choices = [];
+
+          const disabledPlugins = Array.from(plugins.plugins.values()).filter((plugin) =>
+            plugins.getDisabled().includes(plugin.manifest.id),
+          );
+
+          const disabledThemes = Array.from(themes.themes.values()).filter((theme) =>
+            themes.getDisabled().includes(theme.manifest.id),
+          );
+
+          choices.push(
+            ...disabledPlugins.map((plugin) => ({
+              name: plugin.manifest.name,
+              displayName: `Plugin: ${plugin.manifest.name}`,
+              value: plugin.manifest.id,
+            })),
+          );
+          choices.push(
+            ...disabledThemes.map((theme) => ({
+              name: theme.manifest.name,
+              displayName: `Theme: ${theme.manifest.name}`,
+              value: theme.manifest.id,
+            })),
+          );
```

Comment:
Can you sort these alphabetically within the types? Still keep plugins and themes separate but the names within that should be sorted.